client/src/components/DataFiles/DataFilesModals/DataFilesLinkModal.module.scss

```diff
@@ -7,11 +7,14 @@
 }
 
 .help-text {
-  color: #AFAFAF;
+  margin-top: 4px;
+  color: #888F96;
```

Comment:
Everything looks right, but some or all of these styles (whether from original PR #265 or this PR #291) are recreations of an existing style. I'll come back when I can* narrow it down.

_\* To figure out which code produces the existing similar styles, I need JIRA (FP-740 has mention of the class name) **or** production Portal (Manage Account uses a third-party component to render help text) **or** dev Portal (my env is tied up with a hairy incomplete merge conflict)._

Reply:
I found the __existing__ help text styling (from earlier in 2020).

These should be used instead of the new styles (from PR #265 or this PR #291).

__A.__ ★ Use the `_common/Form/FormField` component.
- _Example: [`AppForm.js` via `FormField`'s `description` prop](https://github.com/TACC/Frontera-Portal/blob/master/client/src/components/Applications/AppForm/AppForm.js#L451-L455)_

__B.__ ★ Use the `FormText` component.
- _Sample: `<FormText className="form-field__help" color="muted">`_
- _Example: [`FormField.js`](https://github.com/TACC/Frontera-Portal/blob/master/client/src/components/_common/Form/FormField.js#L78-L80)_

__C.__ __⚠︎__ Use the `form-field__help form-text text-muted` classes directly.
- _Source: [`FormField.js`](https://github.com/TACC/Frontera-Portal/blob/master/client/src/components/_common/Form/FormField.js#L78-L80)_

Reply:
Using FormText component.